src/pages/clients.tsx

```diff
@@ -35,11 +40,71 @@ const ClientsPage: React.FC = () => {
             (major parts of) their web applications.
           </LargeText>
           <CaptionText>Not just a set of banners or a microsite.</CaptionText>
-          <CardGridContainer>TODO: CardGrid</CardGridContainer>
         </GridItem>
       </Grid>
+      <ImageCardGrid>
+        <GridItem xs={12} sm={4}>
+          <InternalLink to="/">
+            <ImageCard
+              alt="Lewandowski"
+              image={data.imagePlaceholder}
+              largeTitle="Client One"
+            />
+          </InternalLink>
+        </GridItem>
+        <GridItem xs={12} sm={4}>
+          <InternalLink to="/">
+            <ImageCard
+              alt="Lewandowski"
+              image={data.imagePlaceholder}
+              largeTitle="Client Two"
+            />
+          </InternalLink>
+        </GridItem>
+        <GridItem xs={12} sm={4}></GridItem>
```

Comment:
Are these the empty tiles? I would put them in a component. This markup doesn't make it clear, tbh.
```javascript
const EmptyGridItem: React.FC = () => (
  <GridItem xs={0} sm={4} />
);
```
And use it in the specific locations. 😇